.github/workflows/wasm.yml

```diff
@@ -16,14 +15,12 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v3
 
-      - name: Setup rust toolchain
-        uses: dtolnay/rust-toolchain@master
+      - name: Setup Nix
+        uses: cachix/install-nix-action@v20
         with:
-          toolchain: 1.66.0
+          nix_path: nixpkgs=channel:nixos-22.11
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install wasm-pack
-        run: cargo install wasm-pack
-
-      - name: Build wasm crate
-        working-directory: ./crates/wasm
-        run: ./build-wasm
+      - name: Build wasm package
+        run: |
+          nix build -L .#wasm
```

Comment:
building this will produce output as listed by command below:

`ls result/ -R`                                                                

```
result/:
README.md  nodejs  web

result/nodejs:
noir_wasm.d.ts  noir_wasm.js  noir_wasm_bg.wasm  noir_wasm_bg.wasm.d.ts

result/web:
noir_wasm.d.ts  noir_wasm.js  noir_wasm_bg.wasm  noir_wasm_bg.wasm.d.ts
```

it's not going to be usable w/o `package.json` that was created in `installPhase.sh`

Reply:
In the install phase, you can copy other files, such as a committed `package.json` into the `$out`